### PR TITLE
Add many entries to the 2019 list.

### PR DESCRIPTION
**Pull request by dklong-csu**

### CODE REVIEW

**Review by bangerth — APPROVED** [review]

**Comment by bangerth** [comment]
pdflatex  is unhappy, but I fail to see what error triggers this. Does someone have an idea by looking at the CI logs?

**Comment by tjhei** [comment]
```
Overfull \hbox (0.25912pt too wide) in paragraph at lines 61--61
\T1/cmr/m/n/9 Magneto-Mechanics: A Nu-mer-i-cal Com-par-i-son of Es-tab-lished 
Hyper-Reduction
[113]
! Undefined control sequence.
<recently read> \uplambda 
                          
l.61 \addnewbibyear{2019}
```

File: publications-2019.bib [hunk 1]
```diff
@@ -1253,4 +1253,636 @@ @misc{engwer2019dynamic
...
+  volume = {75},
+  number = {2},
+  pages = {137--147},
+  author = {K. S. Han and B. H. Park and A. Y. Aydemir and J. Seol},
+  title = {The {DTEQ} Code for Toroidal {MHD} Equilibria with Diamagnetic Current Modeling Using the deal.{II} Finite Element Library},
+  journal = {Fusion Science and Technology}
+}
+
+
+@article{Bastian2019,
```
Comment (tjhei) [inline publications-2019.bib:1647]: This does not use deal.II.

**Comment by bangerth** [comment]
I fixed the lambda issue and removed the entry @tjhei flagged. I also fixed the merge conflict. Let's see what happens!

**Comment by bangerth** [comment]
Seems to have worked.

**Pull request merged**